README.md

```diff
@@ -1,2 +1,20 @@
-# HW01
-Edit the readme
+HW01
```

Comment:
This line looks like a  header in the preview, but you did not add the "#". I am a little bit curious about how you do so?